test/params_test.exs

```diff
@@ -89,14 +89,17 @@ defmodule ParamsTest do
   end
 
 
-  test "changes gets casted values" do
+  test "to_map gets map of struct except for _id" do
+    # This test fails because of a missing @schema
     params = %{
-      "origin" => %{
-        "latitude" => "12.2",
-      }
+      "latitude" => 12.2,
+      "longitude" => 13.3
     }
-    changes = Params.changes BusParams.from(params)
-    assert %{origin: %{latitude: 12.2}} = changes
+    result = params
+              |> LocationParams.from
+              |> Params.to_map
+
+    assert result == %{latitude: 12.2, longitude: 13.3}
   end
```

Comment:
This test fails here: https://github.com/vic/params/blob/master/lib/params.ex#L108-L110
Apperently because there is no `@schema` defined in the module. I'm not sure if it _should_ be defined or not?